src/app/core/services/feature-gate.ts

```diff
@@ -21,6 +21,7 @@ import {catchError, shareReplay, switchMap} from 'rxjs/operators';
 
 export interface FeatureGates {
   konnectivityService?: boolean;
+  OIDCKubeCfgEndpoint?: boolean;
```

Comment:
Are you sure that the API has been updated to return this?

Reply:
https://github.com/kubermatic/kubermatic/pull/9034/files#diff-997ca39511bbd531bc6ed68d834e78ce8194439f367a1cf0130e0748d3033f15R647

Reply:
For the future please link any blocking external PRs and hold the PR so that we won't merge anything too early.